integration-testing/src/mavenTest/kotlin/MavenPublicationMetaInfValidator.kt

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2016-2021 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
```

Comment:
Sadly, I don't know what would be right here, but I'm sure it's not this. My bet is "Copyright 2023".